From 0f3d932337dd753868913687f35b3517e26589c9 Mon Sep 17 00:00:00 2001
From: Andreas Dinter <andreas.dinter@systec-electronic.com>
Date: Mon, 11 Nov 2024 08:30:35 +0100
Subject: [PATCH] configs/am62x_evm_a53_defconfig: Disable SD card voltage
 selection

sysWORXX ctr-xxx base boards do not have the necessary GPIO's to select
SD card voltage.

Signed-off-by: Andreas Dinter <andreas.dinter@systec-electronic.com>
---
 configs/am62x_evm_a53_defconfig | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configs/am62x_evm_a53_defconfig b/configs/am62x_evm_a53_defconfig
index 34e598b519c..0a72f674987 100644
--- a/configs/am62x_evm_a53_defconfig
+++ b/configs/am62x_evm_a53_defconfig
@@ -91,8 +91,6 @@ CONFIG_DM_MAILBOX=y
 CONFIG_K3_SEC_PROXY=y
 CONFIG_FS_LOADER=y
 CONFIG_SUPPORT_EMMC_BOOT=y
-CONFIG_MMC_IO_VOLTAGE=y
-CONFIG_SPL_MMC_IO_VOLTAGE=y
 CONFIG_MMC_UHS_SUPPORT=y
 CONFIG_SPL_MMC_UHS_SUPPORT=y
 CONFIG_MMC_HS200_SUPPORT=y
-- 
2.48.1

